lib/twilio-ruby/rest/ip-messaging/credentials.rb

```diff
@@ -0,0 +1,22 @@
+module Twilio
+  module REST
+    module IpMessaging
+      class Credentials < NextGenListResource
+        def initialize(path, client)
+          @path, @client = path, client
+          @submodule = :IpMessaging
+        end
+      end
+
+      class Credential < InstanceResource
+        def initialize(path, client, params={})
+          super
+          @submodule = :IpMessaging
+          resource :channels,
```

Comment:
I also made the changes for edge

https://github.com/twilio/ruby-private/pull/37